lumapps_api_client/lib.py

```diff
@@ -228,6 +265,9 @@ def token(self, v):
 
     @property
     def service(self):
+        """ Setup the service object.
+            It construct the methods specific to the given API.
```

Comment:
"It construct the methods specific to the given API." doesn't mean much... perhaps just don't put it?